crawl-ref/source/show.cc

```diff
@@ -432,6 +432,8 @@ static void _update_monster(monster* mons)
     {
         mons->ensure_has_client_id();
         monster_info mi(mons);
+        if ( you.can_see(*mons) )
+            mi.turns = you.num_turns;
```

Comment:
I don't really understand how `turns` is supposed to work, sorry. The PR describes this as "the turn number when you first saw this monster", but monster info is regenerated all the time - it doesn't persist when monsters move, etc.

Is the intent that `turns` is the turn at which you *last* saw a monster *at a given location*? I'm moderately sure that's what is implemented here, though I haven't tested.

Reply:
Yes, sorry, the PR description was wrong -- you've got it right above, it's the last time you saw the monster at that location.  I updated the PR description to be more accurate.

Reply:
Thanks! Will take another look at this by early 2025 at the latest.